hazelcast/src/test/java/com/hazelcast/config/MapConfigTest.java

```diff
@@ -1,238 +1 @@
-/*
```

Comment:
Actually with check-style parameter build is failing but this file was changed before me and it was excluded from jacoco-coverage -- https://github.com/hazelcast/hazelcast/commit/4585c99f97c2719f6db46d5c3528d3714e301a40 

Maybe it's a reason why it's failing but the question is - should I change this file or not?!